tensorboard/pip_package/setup.py

```diff
@@ -66,6 +69,10 @@ def get_readme():
         'tensorboard.plugins.beholder': [
             'resources/*',
         ],
+        'tensorboard.plugins.projector': [
+            'tf_projector_plugin/index.js',
```

Comment:
Hmm; this is a little scary. Can we at least add a comment here that
this MUST be kept in sync with the srcs for the `projector_assets` web
library in Bazel? As is, it seems too easy to add a file dependency that
works fine in dev but is omitted in the built Pip package.